portfolio/src/main/java/com/google/sps/servlets/LoadCommentServlet.java

```diff
@@ -80,20 +101,29 @@ public int getNumberOfComments(HttpServletRequest request) throws IOException {
     return selectedNumberOfComments;
   }
 
+  // Get the language into which the comments will be translated
+  public String getLanguageCode(HttpServletRequest request) throws IOException {
+    String languageCode = request.getParameter("language_code");
+
+    return languageCode;
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     UserService userService = UserServiceFactory.getUserService();
 
-    if(!userService.isUserLoggedIn()) {
+    if (!userService.isUserLoggedIn()) {
       response.sendRedirect(REDIRECT_URL);
       return;
     }
     
     numberOfComments = getNumberOfComments(request);
+    languageCode = getLanguageCode(request);
     loadComments();
-
+  
     /** Send Get response to the wepage */
-    response.setContentType("application/json;");
+    response.setContentType("application/json; charset=UTF-8");
+    response.setCharacterEncoding("UTF-8");
```

Comment:
Just checking both these are necessary if you're setting charset in ContentType, still need to set encoding?

Reply:
This code is actually taken from the translation API walk through and it seems to be a recommendation when trying to translate in some languages.